common/source/docs/common-fettec-onewire.rst

```diff
@@ -0,0 +1,147 @@
+
+- :ref:`SERIAL4_PROTOCOL<SERIAL4_PROTOCOL>` = 38 (FETtec_OneWire)
+- :ref:`SERIAL4_OPTIONS<SERIAL4_OPTIONS>` = 0 (default, full duplex)
+
+
+This digital protocol can use the full 1000 to 2000 range. Do set these parameters as well:
+
+- :ref:`MOT_PWM_MAX<MOT_PWM_MAX>` = 2000
+- :ref:`MOT_PWM_MIN<MOT_PWM_MIN>` = 1000
+
+
+Change SERVOx_FUNCTION parameters according to the number of motors and their order. e.g. for a quadcopter:
+
+- :ref:`SERVO1_FUNCTION<SERVO1_FUNCTION>` = 33
+- :ref:`SERVO2_FUNCTION<SERVO2_FUNCTION>` = 34
+- :ref:`SERVO3_FUNCTION<SERVO3_FUNCTION>` = 35
+- :ref:`SERVO4_FUNCTION<SERVO4_FUNCTION>` = 36
+
+FTW Mask:
+After that you must set the servo output channels you want to use in the :ref:`SERVO_FTW_MASK<SERVO_FTW_MASK>` parameter, and again restart your autopilot. For example, a quadcopter using the first four motors would need the first four bits of the bitmask set:
+
+:ref:`SERVO_FTW_MASK<SERVO_FTW_MASK>` = 15
+
+Each bit of the mask describes whether a motor is attached and should be used with oneWire. The value is configured decimal and is the sum of the decimal equivalent of each set bit. Bit 1 corresponds to 2^1 = 1, Bit 2 corresponds to 2^2 = 2,... . E.g. if you have connected Motor 1,2,3,4 the mask is 1+2+4+8 = 15.  
+
+Simply sum the decimal values for the motors connected or read it from the list below.
+
++------------+----+
+|Motor Count |Mask|
++------------+----+
+|1           |1   |
++------------+----+
+|2           |3   |
++------------+----+
+|3           |7   |
++------------+----+
+|4           |15  |
++------------+----+
+|5           |31  |
++------------+----+
+|6           |63  |
++------------+----+
+|7           |127 |
++------------+----+
+|8           |255 |
++------------+----+
+|9           |511 |
++------------+----+
+|10          |1023|
++------------+----+
+|11          |2047|
++------------+----+
+|12          |4095|
++------------+----+
+
+FTW RVMask:
+After that you must set the servo output channels motor rotation direction in the :ref:`SERVO_FTW_RVMASK<SERVO_FTW_RVMASK>` parameter. To leave to motors with their default rotation direction do:
+
+:ref:`SERVO_FTW_RVMASK<SERVO_FTW_RVMASK>` = 0
+
+Each bit of the mask describes whether a motor is rotation direction should be reversed. The value is configured decimal and is the sum of the decimal equivalent of each set bit. Bit 1 corresponds to 2^1 = 1, Bit 2 corresponds to 2^2 = 2,... .  
+
+Optional features can be configured using the instructions at their respective wiki page:
+
+- dynamically change the center frequency of the :ref:`notch filters used to reduce frame vibration noise in the gyros<common-imu-notch-filtering#dynamic-harmonic-notch>`
+- measure :ref:`battery voltage, current and power consumption<common-dshot-blheli32-telemetry#use-as-battery-monitor>`
+
+The :ref:`FETtec ESC configurator <https://github.com/FETtec/ESC-Configurator/releases>` can change all ESC's settings and do ESC firmware updates.
```

Comment:
this link is malformed....use a double underscore immediately following the backtick